admin-front/src/components/Nfts/Selector/ThumbnailSelector.tsx

```diff
@@ -0,0 +1,49 @@
+import { BooleanInput, FileField, FileInput } from 'react-admin';
```

Comment:
I think that most of `DisplaySelector.tsx` and `ThumbnailSelector.tsx` should be abstracted into a single implementation